pkg/controller/perconaservermongodbbackup/backup.go

```diff
@@ -0,0 +1,110 @@
+package perconaservermongodbbackup
+
+import (
+	"time"
+
+	"github.com/percona/percona-backup-mongodb/pbm"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/backup"
+)
+
+type Backup struct {
+	pbm *backup.PBM
+}
+
+func (r *ReconcilePerconaServerMongoDBBackup) newBackup(cr *api.PerconaServerMongoDBBackup) (*Backup, error) {
+	cn, err := backup.NewPBM(r.client, cr.Spec.PSMDBCluster, cr.Spec.Replset, cr.Namespace)
+	if err != nil {
+		return nil, errors.Wrap(err, "create pbm object")
+	}
+
+	return &Backup{pbm: cn}, nil
+}
+
+func (b *Backup) Start(cr *api.PerconaServerMongoDBBackup) (api.PerconaServerMongoDBBackupStatus, error) {
+	err := b.pbm.SetConfig(cr)
```

Comment:
should we reset config is another backup is not in `BackupStateError` or `BackupStateReady` state?